src/Ed25519VerificationKey2018.js

```diff
@@ -0,0 +1,459 @@
+/*!
+ * Copyright (c) 2018-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import * as env from './env.js';
+import * as util from './util.js';
+import {LDVerifierKeyPair} from 'crypto-ld';
+import * as forge from 'node-forge';
+import * as bs58 from 'bs58';
+import * as semver from 'semver';
+import {promisify} from 'util';
+import {createPublicKey, generateKeyPair, sign, verify} from 'crypto';
+import * as _privateKeyNode12 from './ed25519PrivateKeyNode12.js';
+import * as _publicKeyNode12 from './ed25519PublicKeyNode12.js';
```

Comment:
We usually try to sort these for easier human scanning.

Reply:
Thanks, will sort.